eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -71,6 +82,13 @@ case class Features(activated: Map[Feature, FeatureSupport], unknown: Set[Unknow
     unknownFeaturesOk && knownFeaturesOk
   }
 
+  def initFeatures(): Features = Features(activated.collect { case (f: InitFeature, s) => (f: Feature, s) }, unknown)
+
+  def nodeAnnouncementFeatures(): Features = Features(activated.collect { case (f: NodeFeature, s) => (f: Feature, s) }, unknown)
+
+  // NB: we don't include unknown features in invoices, which means plugins cannot inject invoice features.
```

Comment:
Is there a good reason to not let plugins do that?

Reply:
I think we should allow them to include feature bits in invoices at some point.
But this was already the existing behavior: even before this PR, we never included plugin features in invoices (but it wasn't obvious).
So I added this comment to make it obvious, so that we can fix it in a later PR.